examples/qrecipeSchur.cpp

```diff
@@ -110,18 +110,19 @@ int main( )
     Bounds guessedBounds( NV );
     guessedBounds.init(NV);
     for (int i = 0; i < NV; i++) {
-        guessedBounds.setupBound(i, (SubjectToStatus)0);
```

Comment:
Why activate all constraints? This usually leads to rank deficiency of working set constraint Jacobian.

Reply:
I guess I was trying to obtain a non empty Schur complement starting from iteration 0 for debugging purposes. Anyway, this part of the example has been reverted.